src/biogeochem/VOCEmissionMod.F90

```diff
@@ -933,13 +966,30 @@ function get_gamma_T(t_veg240_in, t_veg24_in,t_veg_in, ct1_in, ct2_in, betaT_in,
     if ( (t_veg240_in > 0.0_r8) .and. (t_veg240_in < 1.e30_r8) ) then 
        ! topt and Eopt from eq 8 and 9:
        topt = co1 + (co2 * (t_veg240_in-tstd0))
-       Eopt = Ceo_in * exp (co4 * (t_veg24_in-tstd0)) * exp(co4 * (t_veg240_in -tstd0))
+       if ( (ivt_in == nbrdlf_dcd_brl_shrub) ) then  ! boreal-deciduous-shrub
+       ! coming from BEAR-oNS campaign willows results
+             Eopt = 7.9 * exp (0.217_r8 * (t_veg24_in-273.15_r8-24.0_r8))
```

Comment:
In general, we need hardwired numbers replaced with variables or parameters that have intuitive names.

Specifically 273.15 is already named tfrz or SHR_CONST_TKFRZ in the model and should be replaced.

Reply:
When you have 288.15, you can name it tfrz + 15._r8 and accordingly for 303.15.